nav_msgs/Odometry odom
std_msgs/Bool is_dropout
std_msgs/Bool is_in_obs
std_msgs/Float32 robot_cov_radius
std_msgs/Float32 coverage_metric
std_msgs/Float32 coverage_metric_diff
